test/Integration/Command/NormalizeCommandTest.php

```diff
@@ -306,7 +306,7 @@ public function normalize(Json $json): Json
     }
 
     /**
-     * @dataProvider providerCommandInvocationAndInvalidIndentSize
```

Comment:
Indent size is not relevant in this test, and the provided argument is never used.